pkgs/development/libraries/pango/default.nix

```diff
@@ -50,6 +39,8 @@ in stdenv.mkDerivation rec {
 
   mesonFlags = [
     "-Dgtk_doc=${if stdenv.isDarwin then "false" else "true"}"
+    # Needs to be disabled explicitly, for some reason
```

Comment:
We enable all auto features by default to be notified when new dependencies are added.

Reply:
Thank you, I'm not familiar with meson.